composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "A Kimai 2 plugin, which allows you to backup your environment with a single click.",
     "homepage": "https://github.com/mxgross/EasyBackupBundle",
     "type": "kimai-plugin",
-    "version": "1.4",
+    "version": "1.5",
     "require": {
         "kimai/kimai2-composer": "*"
```

Comment:
please remove that line, the package is deprecated and archived since ages 😁

Reply:
I thought this is the version of my Bundle, isn't it? If not, where do I define it? Thanks

Reply:
The line ` "kimai/kimai2-composer": "*"` should be removed.
So instead of 
```
    "require": {
        "kimai/kimai2-composer": "*"
    },
```
you can mention the Kimai version and others in require-dev:
https://github.com/Keleo/DemoBundle/blob/1.x/composer.json#L56

or just skip it completely. But the mentioned bundle is not only abandoned, it is incompatible with Composer 2.

Reply:
I hope I got it right now and removed the whole require block because it was empty then.